REFramework reorganization & documentation update

aveizu opened the pull request.

aveizu committed:
Reorganized files in REFramwork folder.

aveizu committed:
Updated REFramework documentation.

lavinianastase reviewed (approved):
Looks good, I ran the project and it works.

swittenberger reviewed (changes requested):
- You need to add a .nuspec file in the root of the directory
- You need to add a .local folder in both pt0 and pt1, with a template.json file

You can take examples for them both from the DU project or the Background Process.

aveizu committed:
added .nuspec file, .local folder and images folder

aveizu committed:
Updated dependencies, version, ProjectUrl

-Updated dependencies to: "UiPath.Excel.Activities": "[2.10.4]", "UiPath.System.Activities": "[21.4.1]","UiPath.UIAutomation.Activities": "[21.4.4]"
-Changed version in .nuspec file to 21.10.0
-Updated ProjectUrl in .nuspec file to: https://docs.uipath.com/studio/docs/robotic-enterprise-framework

swittenberger reviewed (approved):
looking good.
